README.md

```diff
@@ -5,32 +5,41 @@
 Helm is a tool for managing Kubernetes charts. Charts are packages of
 pre-configured Kubernetes resources.
 
-Features:
+Use Helm to...
 
-- Helm now has both a client (`helm`) and a server (`tiller`). The
-  server runs inside of Kubernetes, and manages your resources.
-- Helm's chart format has changed for the better:
-  - Dependencies are immutable and stored inside of a chart's `charts/`
-    directory.
-  - Charts are strongly versioned using SemVer 2
-  - Charts can be loaded from directories or from chart archive files
-  - Helm supports Go templates without requiring you to run `generate`
-    or `template` commands.
-  - Helm makes it easy to configure your releases -- and share the
-    configuration with the rest of your team.
-- Helm chart repositories now use plain HTTP instead of Git/GitHub.
-  There is no longer any GitHub dependency.
-  - A chart server is a simple HTTP server
-  - Charts are referenced by version
-  - The `helm serve` command will run a local chart server, though you
-    can easily use object storage (S3, GCS) or a regular web server.
-  - And you can still load charts from a local directory.
-- The Helm workspace is gone. You can now work anywhere on your
-  filesystem that you want to work.
+- Find and use popular software packaged as Kubernetes charts
+- Package your own applications for Kubernetes
+- Create reproducible builds of your Kubernetes applications
+- Intelligently manage your Kubernetes manifest files
+- Manage releases of Helm packages
+
+## Helm in a Handbasket
+
+Helm is a package manager for your Kubernetes cluster. Think of it like
```

Comment:
package manager for your Kubernetes applications ? helm does not really manage the cluster.